documentation/modules/configuring/proc-resizing-persistent-volumes.adoc

```diff
@@ -6,11 +6,11 @@
 = Resizing persistent volumes
 
 [role="_abstract"]
-You can provision increased storage capacity by increasing the size of the persistent volumes used by an existing Strimzi cluster.
-Resizing persistent volumes is supported in clusters that use either a single persistent volume or multiple persistent volumes in a JBOD storage configuration.
+Persistent volumes used by a cluster can be resized without any risk of data loss, as long as the storage infrastructure supports it. 
+Following a configuration update to change the size of the storage, Strimzi instructs the storage infrastructure to make the change. 
+Storage expansion is supported in Strimzi clusters that use a single persistent volume per-broker or multiple persistent volumes per-broker that use JBOD storage. 
```

Comment:
This is confusing because you can have JBOD storage with a single volume.

```suggestion
Storage expansion is supported in Strimzi clusters that use persistent-claim volumes. 
```